CMakeLists.txt

```diff
@@ -38,7 +38,8 @@ enable_testing()
 ### Project options
 ###
 ## Project stuff
-option(YAML_CPP_BUILD_TOOLS "Enable testing and parse tools" ON)
+option(YAML_CPP_BUILD_TESTS "Enable testing" OFF)
```

Comment:
Why default to off?

Reply:
Assumption that most users just want to build and user the library? This "seems" common in other projects, however I don't really care either way. If you'd prefer to still have them `ON` by default, that's fine also. Let me know either way...

Reply:
I'd like the tests to be on by default; first of all, we'd at least have to update the travis CL execution code if it's off by default, and second, when I check out a library, I often want to run the tests. (I'm trying to keep the CMake config as simple as possible, so if, e.g., someone wants to contribute, they don't have to figure out how to get the tests to build.)

Reply:
WFM: the default is now `ON`.